routing-configurer/src/main/java/no/vegvesen/ixn/federation/ServiceProviderRouter.java

```diff
@@ -236,30 +223,46 @@ private void syncPrivateChannelsWithQpid(Set<PrivateChannel> privateChannels, St
                 }
                 logger.info("Creating queue {}", queueName);
                 VirtualHostAccessController provider = qpidClient.getQpidAcl();
-                provider.addExchangeWriteAccess(name,queueName);
-                provider.addExchangeWriteAccess(peerName,queueName);
-                provider.addQueueReadAccess(name,queueName);
-                provider.addQueueReadAccess(peerName,queueName);
+                provider.addExchangeWriteAccess(name, queueName);
+                provider.addExchangeWriteAccess(peerName, queueName);
+                provider.addQueueReadAccess(name, queueName);
+                provider.addQueueReadAccess(peerName, queueName);
                 qpidClient.postQpidAcl(provider);
                 privateChannel.setStatus(PrivateChannelStatus.CREATED);
+                if (privateChannel.getEndpoint() == null) {
```

Comment:
This is, in effect, the same as asking if the name is set. See the comment on the private channel class